ctx.go

```diff
@@ -288,12 +290,11 @@ func (c ContextNoBody) QueryParam(name string) string {
 		slog.Warn("query parameter not expected in OpenAPI spec", "param", name, "expected_one_of", c.params)
 	}
 
-	_, found := c.Req.URL.Query()[name]
-	if !found {
+	if !c.urlValues.Has(name) {
 		defaultValue, _ := c.params[name].Default.(string)
```

Comment:
It's not a code you are updating.

But I'm worried about a possible panic if c.params[name] is not defined

Reply:
Yeah I agree that this code looks odd for that. I don’t believe it’s actually possible though now with #217. The validation there should handle this.

It would be a bit nice looking to handle it here I guess, but it wouldn’t belong with this PR

Reply:
Yes, it is definitely not related to this PR.

But I raised the point, because it was weird to me

Reply:
Do you plan to open an issue for this?

Reply:
Ehhh, I’m neither here nor there. Again it’s not really possible to panic, but it would make the code look less weird I guess.

Reply:
OK. A crime without victim then 😅😁